src/terra-dev-site/test/ActionHeader.test.mdx

```diff
@@ -0,0 +1,95 @@
+import { Badge } from 'terra-action-header/package.json?dev-site-package';
```

Comment:
Did you intent to pull this in?

Reply:
I left it in as an example of a larger document, but i don't have any tests around it explicitly.

Reply:
sounds good. my only worry would be that we change/remove examples in action header. We won't call those a MVB so could break us